includes/wikia/services/ApiService.class.php

```diff
@@ -112,6 +114,7 @@ private static function getHostByDbName( $dbName ) {
 	 * @return array $options
 	 */
 	public static function loginAsUser() {
+		global $wgEnableHeliosExt;
```

Comment:
Note the `$app = F::app()` in the following line. Using `$app->wg->EnableHeliosExt` later seems more convenient that the `global` reference. And consistent with the rest of the code refering to `wgCookiePrefix` and `wgRequest`.